objects/cash_register_2.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+# Cash Register
+class CashRegister2
+  def initialize
+    cash
```

Comment:
You don't call `cash` anywhere else, so please remove the `cash` method and move the `@total = 0.00` into the `initialize` method.